docs/troubleshooting.rst

```diff
@@ -32,3 +32,19 @@ because one if its dependencies is missing.
 To fix this, you can either install it locally or upgrade your version of pip::
 
   $ pip install --upgrade pip
+
+socket.error: [Errno 48] Address already in use
+===============================================
+Kinto runs on default port 8888 and this error is produced when another service has occupied port 8888
+
+To fix this, see which service is running on port 8888::
+
+$ sudo lsof -i :8888
+
+and kill the process using PID from output::
+
+$ kill -kill [PID]
+
+cliquet.storage.exceptions.BackendError: OperationalError [Postgres Service]
+===========================================================================
+This error is produced when there is a problem to connect to postgres database or postgres service is not running. 
```

Comment:
You could add a way to solve this: "Make sure sure the postgres server is running properly".